pkg/resource/web_acl/sdk.go

```diff
@@ -4365,6 +4399,15 @@ func (rm *resourceManager) sdkUpdate(
 	ko := desired.ko.DeepCopy()
 
 	rm.setStatusDefaults(ko)
+
+	// Check if LoggingConfiguration fields have changed and sync if needed
+	if delta.DifferentAt("Spec.LoggingConfiguration") {
+		// Call the syncLoggingConfiguration function to update the logging configuration
+		err = syncLoggingConfiguration(ctx, rm, desired, latest, delta)
+		if err != nil {
+			return nil, err
+		}
+	}
```

Comment:
this means that if there's only a `Spec.LoggingConfiguration` diff, you're making two API calls, `UpdateWebACL` and `syncLoggingConfig`

Reply:
You probably want to make this one a pre update hook, with something like
```
	if delta.DifferentAt("Spec.LoggingConfiguration") {
		// Call the syncLoggingConfiguration function to update the logging configuration
		err = syncLoggingConfiguration(ctx, rm, desired, latest, delta)
		if err != nil {
			return nil, err
		}
	}
    if !delta.DifferentExcept("Spec.LoggingConfiguration") {
        return
    }
```